x-pack/plugins/synthetics/common/rules/alert_actions.ts

```diff
@@ -134,14 +140,45 @@ function getIndexActionParams(translations: Translations, recovery = false): Ind
       indexOverride: null,
     };
   }
+
+  if (isLegacy) {
+    return {
+      documents: [
+        {
+          monitorName: '{{context.monitorName}}',
+          monitorUrl: '{{{context.monitorUrl}}}',
+          statusMessage: '{{{context.statusMessage}}}',
+          latestErrorMessage: '{{{context.latestErrorMessage}}}',
+          observerLocation: '{{context.observerLocation}}',
+        },
+      ],
+      indexOverride: null,
+    };
+  }
+
+  if (recovery) {
+    return {
+      documents: [
+        {
+          monitorName: '{{context.monitorName}}',
+          monitorUrl: '{{{context.monitorUrl}}}',
+          statusMessage: '{{{context.status}}}',
+          latestErrorMessage: '{{{context.latestErrorMessage}}}',
+          observerLocation: '{{context.locationName}}',
+          recoveryReason: '{{context.recoveryReason}}',
+        },
+      ],
+      indexOverride: null,
+    };
+  }
   return {
     documents: [
       {
         monitorName: '{{context.monitorName}}',
         monitorUrl: '{{{context.monitorUrl}}}',
```

Comment:
@Shahzad if you know why, can you please explain why some of these are `{{{` and some are `{{`

Reply:
this is for url value escaping.